src/Assignments/ProblemIframe.tsx

```diff
@@ -34,7 +34,9 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
     const [renderedHTML, setRenderedHTML] = useState<string>('');
     const [loading, setLoading] = useState(true);
     const [error, setError] = useState('');
+    const [lastSubmission, setLastSubmission] = useState({});
     const height = '100vh';
+    let isClean = false;
```

Comment:
This should be scoped to the function since it is only used in there

Reply:
this is var vs let?

Reply:
Let's remove